content/en/blog/_posts/2023-12-05-kubernetes-1.29.md

```diff
@@ -0,0 +1,182 @@
+
+## What's New in [sig-windows](https://github.com/kubernetes/community/tree/master/sig-windows)
+
+### Alpha: Add support to containerd/kubelet/CRI to support image pull per runtime class
+
+This feature adds support to pull container images based on runtime class. This feature is off by default in v1.29 under a feature gate called `RuntimeClassInImageCriApi`.
+
+Container images can either be a manifest or an index. When the image being pulled is an index (image index has a list of image manifests ordered by platform), platform matching logic is used to pull an appropriate image manifest from the index. By default, the platform matching logic picks a manifest that matches the host that the image pull is being executed from. This can be limiting for VM-based containers where a user could pull an image with the intention of running it as a VM-based container, for example, Windows Hyper-V containers.
+
+The image pull per runtime class feature adds support to pull different images based the runtime class specified. This is achieved by referencing an image by a tuple of (`imageID`, `runtimeClass`), instead of just the `imageName` or `imageID`. Container runtimes could choose to add support for this feature if they'd like. If they do not, the default behavior of kubelet that existed prior to Kubernetes 1.29 will be retained.
+
+### Alpha: Implement In-place Update of Pod Resources
+
+## Graduations, deprecations and removals for Kubernetes 1.29
+
+### Graduated to stable
+
+This release includes a total of 11 enhancements promoted to Stable:
+
+- [Remove transient node predicates from KCCM's service controller](https://github.com/kubernetes/enhancements/issues/3458)
+- [Reserve nodeport ranges for dynamic and static allocation](https://github.com/kubernetes/enhancements/issues/3668)
+- [Priority and Fairness for API Server Requests](https://github.com/kubernetes/enhancements/issues/1040)
+- [KMS v2 Improvements](https://github.com/kubernetes/enhancements/issues/3299)
+- [Support paged LIST queries from the Kubernetes API](https://github.com/kubernetes/enhancements/issues/365)
+- [ReadWriteOncePod PersistentVolume Access Mode](https://github.com/kubernetes/enhancements/issues/2485)
+- [Kubernetes Component Health SLIs](https://github.com/kubernetes/enhancements/issues/3466)
+- [CRD Validation Expression Language](https://github.com/kubernetes/enhancements/issues/2876)
+- [Introduce nodeExpandSecret in CSI PV source](https://github.com/kubernetes/enhancements/issues/3107)
+- [Track Ready Pods in Job status](https://github.com/kubernetes/enhancements/issues/2879)
+- [Kubelet Resource Metrics Endpoint](https://github.com/kubernetes/enhancements/issues/727)
+
+### Deprecations and removals
+
+#### Removal of in-tree integrations with cloud providers ([KEP-2395](https://kep.k8s.io/2395))
```

Comment:
This is pretty big change, thanks for adding it to the deprecations/removal list. I'm wondering if there's more we can do to make this change stand out more since it's a breaking change for some clusters. Can we maybe add a "What's new in sig-cloud-provider?" section at the top that also highlights this? Thoughts @elmiko @sftim (continuing the discussion from https://github.com/kubernetes/website/pull/44010)